_1327/information_pages/templatetags/information_pages_templatetags.py

```diff
@@ -0,0 +1,15 @@
+from django.template import Library
+
+from _1327.information_pages.models import InformationDocument
+
+register = Library()
+
+
+@register.assignment_tag(takes_context=True)
```

Comment:
please register a filter that accesses a new model method `can_user_see_author` instead. have a look at https://github.com/fsr-itse/EvaP/blob/e125a887143848408a5b9c9be77fea53df6d8c77/evap/evaluation/templatetags/evaluation_filters.py#L81 for an example.

you can then use `{% if document.last_author and document|can_user_see_author:request.user %}` in the template.